### DISCUSSION TITLE
xDrip Problems with New Dexcom Transmitters

### DISCUSSION THREAD

**Discussion by enginemr2** [NightscoutFoundation/xDrip#2785] [General]
For the last two Dexcom transmitters I have found they will no longer take a calibration from xDrip. They respond with an error code 12. Also the last two transmitters have disabled the Dexcom receiver. The Serial numbers of the last two transmitters are 8XXXX and 8XXXX (redacted). 
The transmitters do respond with the Blood Glucose values but for any calibration error they respond with the error code 12. I am using a smartwatch as a primary receiver for my smartphone, the same setup I have used for many years.

**Comment by Navid200** [comment]
Please post a screenshot of the system status page.  You can blackout the transmitter ID.
Please post a screenshot of the logs for when you submitted the calibration.
What was the reading at the time you submitted the calibration.

**Reply by enginemr2** [reply]
Right now I have a problem posting a screen shot of the logs. l am writing this problem with my computer so I must transfer the information and screen shot from my smartphone. 
Basically I was trying to calibrate the Dexcom reading of 150 down to the actual value of 125. Even with a much lower percentage change I still get the error code of 12 from the error reported in the log. Using older Dexcom transmitters I have never had this problem. I will work on pasting the logs. Also another new development is the actual Dexcom receiver no longer connects. I suspect this might be linked to the xDrip device connection to port 2 of the transmitter. 
I am a very long-term user of xDrip with Dexcom G5 and G6 devices, never have seen a problem like this.

**Reply by Navid200** [reply]
After creating the screenshot on the phone, email it to yourself.  Then, receive it on your desktop.

**Reply by enginemr2** [reply]
I had already sent the screen shot to the computer via email. I could send either a jpg of png file but can't seem to get a reproduction on this website. 
Interesting dilemma.

**Reply by Navid200** [reply]
While you are editing your post, tap at the bottom border and it let's you attach an image.

**Comment by enginemr2** [comment]
Navid, This is the screen shot of the calibration error code 12 after trying to calibrate. 



    On Sunday, April 16, 2023 at 11:25:40 AM PDT, Navid ***@***.***> wrote:  
 
 


Please post a screenshot of the system status page. You can blackout the transmitter ID.
Please post a screenshot of the logs for when you submitted the calibration.
What was the reading at the time you submitted the calibration.

**Comment by enginemr2** [comment]
Let me know if you want any other information. As I previously mentioned my Dexcom receiver was also disabled from communicating with the Dexcom transmitter. I usually replace the sensor with the Dexcom receiver but will now for the first-time use xDrip to replace the sensor. 
    On Monday, April 17, 2023 at 07:34:32 AM PDT, Dennis McFayden ***@***.***> wrote:  
 
  Navid, This is the screen shot of the calibration error code 12 after trying to calibrate. 



    On Sunday, April 16, 2023 at 11:25:40 AM PDT, Navid ***@***.***> wrote:  
 
 


Please post a screenshot of the system status page. You can blackout the transmitter ID.
Please post a screenshot of the logs for when you submitted the calibration.
What was the reading at the time you submitted the calibration.

**Comment by enginemr2** [comment]
All attempts to calibrate resulted in the same error message. The attempted calibration was anywhere between 5 to 25% with no success. 
    On Monday, April 17, 2023 at 07:37:34 AM PDT, Dennis McFayden ***@***.***> wrote:  
 
  Let me know if you want any other information. As I previously mentioned my Dexcom receiver was also disabled from communicating with the Dexcom transmitter. I usually replace the sensor with the Dexcom receiver but will now for the first-time use xDrip to replace the sensor. 
    On Monday, April 17, 2023 at 07:34:32 AM PDT, Dennis McFayden ***@***.***> wrote:  
 
  Navid, This is the screen shot of the calibration error code 12 after trying to calibrate. 



    On Sunday, April 16, 2023 at 11:25:40 AM PDT, Navid ***@***.***> wrote:  
 
 


Please post a screenshot of the system status page. You can blackout the transmitter ID.
Please post a screenshot of the logs for when you submitted the calibration.
What was the reading at the time you submitted the calibration.

**Reply by jamorham** [reply]
Unfortunately no images come through. You can't send them to github via email you need to edit your entries on the discussion on the website and drag and drop the image in to the text edit box there. Your real name also appears in the email messages that you've sent back on this github discussion. If you want to remove that then you also need to visit the page on github for this discussion to edit and remove that information.

**Comment by Navid200** [comment]
If you follow the instructions, which you can find here https://navid200.github.io/xDrip/, most people will be fine and have no trouble.

It took a lot of convincing for these discussions to be opened up.  My idea was to allow people to ask for help here instead of facebook, so that the developers could see what users have difficulty with.  

But, this is really unacceptable.  Every time you post, the developers get a notification.  You have made 3 posts here just to say that you have uploaded a screenshot.  But, you can see yourself that there is none.  And the first screenshot I asked for was your status page.  Very likely you have enabled engineering mode or something else is not working because you have not followed the instructions probably.

We want to help you.
That's why we spend time here to help you.  Would you please be considerate and put yourself in our shoes?

**Comment by jamorham** [comment]
I think the most likely issue here is that if WearOS is being used as the primary receiver and the phone is not connecting to the transmitter at all then phone based calibrations will likely not work. To fix this, make it so the phone can connect to the transmitter a few times so it can sync the time information but even then that might not fully resolve it. Of course we don't know if its WearOS smartwatch as that information hasn't been provided. I'm just guessing here though.

We need to know:

- xDrip version being used
- Phone model being used
- Smart watch model being used
- Transmitter firmware version from status page

**Comment by enginemr2** [comment]
Navid, Since I am having a problem pasting a screen shot of the Dexcom calibration problem I will have to pass on that attempt. What I need is a step-by-step procedure to remove and install the Dexcom G6 sensor. After all of the years I have used xDrip I have never used it to change a sesnor, always used the Dexcom transmitter. Since the Dexcom receiver is now inoperative unless a transmitter is change, I must use xDrip. I may have used xDrip when using the G5 but don't have any relative data. 


    On Sunday, April 16, 2023 at 11:25:40 AM PDT, Navid ***@***.***> wrote:  
 
 


Please post a screenshot of the system status page. You can blackout the transmitter ID.
Please post a screenshot of the logs for when you submitted the calibration.
What was the reading at the time you submitted the calibration.

**Comment by enginemr2** [comment]
The xDrip version is 2022.11.16The phone model is the Samsung Galaxy S5. The Smart watch is the Sony Smartwatch 3, been using that watch since the Dexcom G5
I don't see the Transmitter firmware version on the xDrip Status page. I have all of the data from the Transmitter box but there is no information on the firmware. I have pasted the transmitter Serial Number but I guess that is not allowed. 
    On Monday, April 17, 2023 at 08:22:30 AM PDT, JamOrHam ***@***.***> wrote:  
 
 


I think the most likely issue here is that if WearOS is being used as the primary receiver and the phone is not connecting to the transmitter at all then phone based calibrations will likely not work. To fix this, make it so the phone can connect to the transmitter a few times so it can sync the time information but even then that might not fully resolve it. Of course we don't know if its WearOS smartwatch as that information hasn't been provided. I'm just guessing here though.

We need to know:
   
   - xDrip version being used
   - Phone model being used
   - Smart watch model being used
   - Transmitter firmware version from status page

**Reply by Navid200** [reply]
You need to set up xDrip on your phone to be the collector.  This means, you need to at least temporarily, stop using the watch as a collector and start using your phone for that.  Then, the status page will show everything needed for us to be able to help you.

Take a screenshot of the G5/G5 status page.  It looks something like this:  https://navid200.github.io/xDrip/docs/StatusG5G6.html

Find a way to provide the screenshot to us.  You can use Google Drive, Google photo, dropbox, facebook, any image sharing platform.

When we can see the status page, we will help if we can.

and upload it to anywhere you like, facebook, Google Drive,
